packages/webui/src/context/ApplicationContext.test.tsx

```diff
@@ -582,7 +633,7 @@ describe('ApplicationContext', () => {
       });
 
       const valueAfter = getByTestId('value');
-      expect(valueAfter).toHaveTextContent('2');
+      expect(valueAfter).toHaveTextContent('1');
```

Comment:
💬 The change to how mock traces were organised meant that IDs were out of sequence, and subsequently meant that some tests were not exercising code fully.  For example, a test which previously attempted to move to a previous trace wouldn't execute the code to go to the previous trace, because the currently selected trace had now become the first one.

A change in the mock data now ensures that traces will get sequential IDs, regardless of how they are organised.